### ISSUE TITLE
bug api test verbosity level option

### ISSUE THREAD

**Issue by MontrealSergiy** [aces/cbrain#1198]
A command from README fails

```
[sboroday@ace-vh-2 BrainPortal]$ rake cbrain:test:api:client -v4 groups/list
invalid option: -4
```

missing space, perhaps?

**Comment by prioux** [comment]
Investigate this, and fix the doc if necessary

**Comment by MontrealSergiy** [comment]
@prioux yes adding space fixes the issue (did not test for macs). If it is ok I will include in PR for offline bourreaux task creation

**Comment by MontrealSergiy** [comment]
docs are corrected in pr #1200
